utils/grafana-container.bash

```diff
@@ -0,0 +1,12 @@
+set -ex
+docker rm -f grafana || true
+docker run -p 3000:3000 \
```

Comment:
i've added a `docker pull grafana/grafana-oss:latest` to my script, maybe we should add it here, too?